src/crypto/bitcoincash.ts

```diff
@@ -59,7 +58,8 @@ class BitcoinCash implements ICrypto {
 
   // eslint-disable-next-line class-methods-use-this
   validateAddress(address: string): boolean {
-    return fallbackValidateAddress(address);
+    const { isValidAddress } = bchaddr;
+    return isValidAddress(address);
```

Comment:
I've seen this construct in lots of places, and I don't understand why it's used. Why
```
    const { isValidAddress } = bchaddr;
    return isValidAddress(address);
```
and not
```
    return bchaddr.isValidAddress(address);
```
Is there some advantage with the former? Just curious.

Reply:
fixed